### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by mimansha-swarup**
https://user-images.githubusercontent.com/69811810/159273369-454994b7-4c04-432f-9776-e7dd10cda6f2.mp4

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,42 +1,16 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Navbar } from "./Component/index";
+import { BrowserRouter } from "react-router-dom";
```
Comment (KaustubhM1997) [inline src/App.js:3]: you can directly import Browser Router to index.js

File: src/Component/index.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import Navbar from "./Navbar/Navbar";
```
Comment (KaustubhM1997) [inline src/Component/index.js:1]: Great work!

**Review by KaustubhM1997 — APPROVED** [review]

File: src/Component/Footer/Footer.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+      <div className="footer-actions-container">
+
+      
+      <div class="footer-actions">
+       
+          <h6 class="headline6">About</h6>
+          <ul>
+            <li>
+              <a href="/">About Us</a>
```
Comment (Swapnilkota108) [inline src/Component/Footer/Footer.js:34]: Routes???

Comment (mimansha-swarup) [reply src/Component/Footer/Footer.js:34]: For now its dummy  will add later

**Pull request merged**